app/main.py

```diff
@@ -1,3 +1,28 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer
+from app.car import Car
+
+
+def shop_trip() -> None:
+    with open("app/config.json") as file:
```

Comment:
Make sure you write the correct path to the JSON file. Verify that the path 'app/config.json' is correct and accessible from this script.